docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.7'
+
+services:
+  postgres:
+    image: bitnami/postgresql:latest
```

Comment:
It was an "automatic" choice. I have been using Bitnami in all of my projects because they have a lot of security features built in and are up to date. But I have no opposition to changing it.

Reply:
I think it will be easier for other developers to use it if we do not need another third party installation. But I am open to discussion if you think otherwise. Thanks.

Reply:
I think we can go with the Bitnami version since it has the extra security and performance features and it's just as easy to use as the standard version.